src/common/contracts.ts

```diff
@@ -13,11 +14,31 @@ type ContractAddressMap = {
 };
 
 const contractAddresses: ContractAddressMap = {
-	[Network.Mainnet]: {
-		vesting: process.env.REACT_APP_VESTING_CONTRACT,
-	},
 	[Network.Kovan]: {
-		vesting: '0x1d308f379c94e668Ef90328bF0C11dFFbd5055E9',
+		name: [
```

Comment:
This format doesn't work, please make each vesting contract an object, instead of 3 arrays which has different elements of a vesting contract.